release-notes.md

```diff
@@ -1,5 +1,14 @@
 # Mercury 13.20.0 #
 
+## Media Repurposing private content bug ##
+- Issue Type: Task
+- Issue ID: PLAT-8770
+
+### Configuration ###
+	None.
+### Deployment scripts ###
+	Index category and entry tables to sphinx
```

Comment:
Add the file paths to the populate scripts so it will be known what needs to be executed.
For example: 
php /opt/kaltura/app/deployment/base/scripts/populateSphinxCategories.php